MatrixSDK/Contrib/Swift/Data/MXSessionEventListener.swift

```diff
@@ -0,0 +1,20 @@
+//
+//  MXSessionEventListener.swift
+//  MatrixSDK
+//
+//  Created by Avery Pierce on 7/12/17.
+//  Copyright © 2017 matrix.org. All rights reserved.
```

Comment:
Unfortunately, matrix.org is not yet a foundation, nor a legal entity. Could you please remove this copyright line?
For consistency, would it be possible to use the same licence and file header as you did before (like https://github.com/matrix-org/matrix-ios-sdk/blob/6201d5709a634f7c0ea85679c04d12bf582dedab/MatrixSDK/SwiftMatrixSDK.h)?